controllers/rollingupgrade_controller.go

```diff
@@ -375,7 +375,8 @@ func (r *RollingUpgradeReconciler) SetStandby(ruObj *upgrademgrv1alpha1.RollingU
 
 	instanceState, err := getGroupInstanceState(asg, instanceID)
 	if err != nil {
-		return err
+		r.info(ruObj, "Cannot get instance state ", "instanceID", instanceID)
```

Comment:
Also, will the only reason that `getGroupInstanceState` returns an error be that the instance is gone? We might want to do a more specific check, no?

Reply:
Yes. The only instance when `getGroupInstanceState` errors out is with a NotFound.
```
func getGroupInstanceState(group *autoscaling.Group, id string) (string, error) {
    for _, instance := range group.Instances {
        if aws.StringValue(instance.InstanceId) != id {
            continue
        }
        return aws.StringValue(instance.LifecycleState), nil
    }
    return "", errors.Errorf("could not get instance group state, instance %v not found", id)
}
```

Reply:
In this case, I would put the message as `Warning` and log the original error message which already says that instance could not get the state. The implementation shouldn't swallow the error, just push it as warning.

Reply:
+1 to what @kevdowney said